pages/cloud/storage/nas_nfs/guide.en-gb.md

```diff
@@ -1,236 +1,194 @@
+- [A partition created on the service with the NFS protcol enabled](https://docs.ovh.com/gb/en/storage/nas/get-started/#create_partition)
+- [An ACL entry for the server's IP address](https://docs.ovh.com/gb/en/storage/nas/get-started/#access_control)
+- Administrative access (root) to your server via SSH or GUI
 
-```sh
-nfs-client install capability
-```
+## Instructions
 
-Then use the following mount command:
+The following sections contain configuration examples for the most commonly used distributions / operating systems. The first step is always to log in to your server via SSH or connecting to the GUI of yor installed OS. The examples below presume you are logged in as a user with elevated permissions.
 
+You will also need the **internal name** and the **IP address** of your HA-NAS service which you can find in the email received after the installation or in your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).
 
-```sh
-mount -t nfs -o _netdev,mountproto=tcp IP_NAS:/NFS_PATH /MOUNTING_FOLDER
-```
+The following notations are used as arguments in the command line sections below. Replace them with the appropriate values when entering the commands.
 
 |Argument|Description|
 |---|---|
-|IP_NAS|Corresponds to NAS name or IP|
-|/NFS_PATH|Path to NFS server for sharing (Ex: "nas-000YY/partition")|
-|MOUNTING_FOLDER|Corresponds to the folder where you will mount your NFS share on your server|
+|IP_HA-NAS|The IP address of the HA-NAS (Example: 10.1.1.1)|
+|NFS_PATH|The path to the HA-NAS partition to mount, consisting of the service name and your partitions' name (Example: zpool-123456/partition01)|
+|MOUNTING_FOLDER|The local folder for your mounted partition|
 
 
-> [!primary]
+> [!warning]
 >
-> In order to automate the NAS mount when your distribution starts up, add the following line to the /etc/fstab file:
+> The NFS user is `root`, rights changes with this user can generate conflicts with existing CIFS/SMB rights.
 > 
-> ```
-> IP_NAS:/NFS_PATH /MOUNTING_FOLDER nfs rw,_netdev,mountproto=tcp 0 0
-> ```
->
 
-*For example:*
 
-```sh
-mount -t nfs -o _netdev,mountproto=tcp 10.16.XXX.YYY:zpool-999888/PartitionName /media/NasHA -v
-```
-
-|Argument|Description|
-|---|---|
-|IP_NAS|10.16.XXX.YYY|
-|/NFS_PATH|zpool-999888/PartitionName|
-|MOUNTING_FOLDER|/media/NasHA -v|
-
-### CentOS 
-
-Compatibility: CentOS
-
-To mount an NFS share under CentOS, you must:
-
-- Connect to the server in SSH.
-- Install the "nfs-utils" and "rpcbind" packages via the command:
+### Debian / Ubuntu
 
+Install the package `nfs-common`:
 
-```sh
-yum install nfs-utils rpcbind
+```bash
+ubuntu@server:~$ sudo apt install nfs-common
 ```
 
-Then restart the `rpcbind` service with the following command:
-
+Then use the following mount command:
 
-```sh
-/etc/init.d/rpcbind start
+```bash
+ubuntu@server:~$ sudo mount -t nfs -o _netdev,mountproto=tcp IP_HA-NAS:NFS_PATH /MOUNTING_FOLDER
```

Comment:
I seriously doubt the necessity of these options:
- do you have explanations for those?
- why we need to use it in a _standard use case_?